dspace-api/src/test/data/dspaceFolder/config/local.cfg

```diff
@@ -105,3 +105,34 @@ plugin.sequence.java.util.Collection = \
     java.util.LinkedList, \
     java.util.Stack, \
     java.util.TreeSet
+
+#### OpenSearch Settings ####
+# NB: for result data formatting, OpenSearch uses Syndication Feed Settings
+# so even if Syndication Feeds are not enabled, they must be configured
+# enable open search
+websvc.opensearch.enable = true
```

Comment:
Are you certain that we need all of the `websvc.opensearch.*` settings copied into the `local.cfg` for the Unit Test framework?  The Unit Test framework should default to using the default settings in `dspace.cfg`...and most of these look to just be the default settings: https://github.com/DSpace/DSpace/blob/master/dspace/config/dspace.cfg#L1336-L1361

While I haven't tried it, I believe you should be able to remove all of these settings *except* for setting `websvc.opensearch.enable = true` (as the default is `false`). 

You may need to update a few unit tests if they are specifically looking for `DS7` or `DSpace 7` in the `shortname`, `longname` or `description`.  But, you could change them to instead look for `DSpace at My University`, which is the default value of `dspace.name`: https://github.com/DSpace/DSpace/blob/master/dspace/config/dspace.cfg#L42

Reply:
done